roles/1-prep/tasks/detected_network.yml

```diff
@@ -250,7 +250,7 @@
 - name: 2 or more devices on the LAN - use bridging
   set_fact:
      xsce_lan_iface: br0
-  when: 'discovered_lan_iface != "none" and num_lan_interfaces >= "2"'
+  when: 'discovered_lan_iface != "none" and num_lan_interfaces >= "1"'
 
```

Comment:
I'd rather see a new stanza with "when is_debian" rather than mess with the CentOs' code path where is could break working code.